lib/opsbot.js

```diff
@@ -47,13 +48,12 @@ module.exports.sendMessageAsync = util.promisify((msg, callback) =>
  * @param callback Function that is called after the message is sent.
  * Called with callback(err, response, body)
  */
-module.exports.sendSlackAlmaMessage = (msg, channel, options, callback) => {
-  // Fall-back to the old token config if it exists
-  const token = config.secretSlackToken || config.secretSlackProctorToken;
+module.exports.sendSlackMessage = (msg, channel, options, callback) => {
```

Comment:
The word "Alma" is a remnant of the early days of this Slack code, when it wrote messages as a user named "Alma". It's not necessary to include that in function names.